docs/img/src/architecture.puml

```diff
@@ -0,0 +1,39 @@
+@startuml
+skinparam componentStyle rectangle
+
+package "OpenTelementry connector" {
```

Comment:
I know there is Opentelemetry Connector and Collector, but maybe Collector is a better term to use here since it represents the whole long running process.